server/src/main/java/com/unosquare/carmigo/service/PlatformUserService.java

```diff
@@ -35,6 +39,21 @@ public class PlatformUserService {
   private final EntityManager entityManager;
   private final BCryptPasswordEncoder bCryptPasswordEncoder;
 
+  public void confirmEmail(final String email) {
+    final Optional<PlatformUser> platformUserOptional = platformUserRepository.findPlatformUserByEmail(email);
+    if (platformUserOptional.isPresent()) {
+      if (!platformUserOptional.get().getUserAccessStatus().getStatus().equals(ACTIVE)) {
+        platformUserOptional.get()
+            .setUserAccessStatus(entityManager.getReference(UserAccessStatus.class, ACTIVE_USER_STATUS));
+        platformUserRepository.save(platformUserOptional.get());
+      } else {
+        throw new IllegalStateException("User is already active");
```

Comment:
Perhaps put the string in this error message into a constant field within the class?